config/initializers/sufia.rb

```diff
@@ -208,7 +208,8 @@
      "2014-06" => "2014-06",
      "2014-11" => "2014-11",
      "2015-06" => "2015-06",
-     "2015-11" => "2015-11"
+     "2015-11" => "2015-11",
+     "2016-06" => "2016-06"
```

Comment:
I don't feel right about hiding unrelated changes in biggish commits.  If you don't feel it warrants a new PR, at least make it a separate commit so we can easily see the history.